Exercise_A/index.html

```diff
@@ -17,29 +17,225 @@
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <aside class="bg-dark">
+      <div class="container">
+        <div class="WOOT">
+          <h2>Seriously, It's This Fucking Template!</h2>
+        </div>
+        <a href="http://google.com" class="btn btn-secondary btn-xl rounded-pill pages-scroll">GOOGLE THAT SHIT</a>
+      </div>
+      <br>
+      <br>
+      <br>
+      <br>
+    </aside>
     <!-- contact me container -->
+    <section id="contact-me">
+      <div class="container-fluid">
+        <div>
+          <h2>
+            My Contact Information
+          </h2>
+          <hr style="height: 3px; width: 50px; margin: auto;" class="hr2">
+        </div>
+        <div class="mx-auto" style="width: 50%;">
+          <p class="random-words-lol">
+            Here's my phone number and email address, because hey that's part of the template and why would you not include that on your webpage? You'd be a fool not to! I can't wait for all the job offers to come pouring in from all the companies wanting my insane website design skills.
+          </h4>
+          <br>
+          <p class="random-words-lol">Honestly this template does look really nice, though.</p>
+        </div>
+        <div class="phones-n-shit">
+          <div class="row justify-content-center">
+            <div class="col-lg-4 col-lg-offset-2 text-center">
+              <i class="fa fa-phone fa-3x wow bounceIn" style="visibility: visible; animation-name: bounceIn"></i>
+              <p>+98811969</p>
+            </div>
+            <div class="col-lg-4 col-lg-offset-2 text-center">
+              <i class="fa fa-envelope fa-3x wow bounceIn" style="visibility: visible; animation-name: bounceIn"></i>
+              <p>holy@shittt.com</p>
+            </div>
+          </div>
+        </div>
+        
+        
+      </div>
+
+    </section>
 
     <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.1.3/js/bootstrap.min.js" integrity="sha512-OvBgP9A2JBgiRad/mM36mkzXSXaJE9BEIENnVEmeZdITvwT09xnxLtT4twkCa8m/loMbPHsvPl0T8lRGVBwjlQ==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
+
 
     <!-- 
       All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
 
       If you do not use CSS to do smooth scroll then... You need the CDN for Smooth Scroll
       Your custom JavaScript
      -->
+
+     
   </body>
 </html>

```

Comment:
Great job on the website, though I would suggest to change the icons that you have on the website. Also the Wow.js does not load on your page. Try your best to not use as much CSS as possible and use the Built-In Bootstrap Classes. For the Animate, to have it run you would need to add the Javascript and CSS links. You can refer to the documentation here 

Link: [https://wowjs.uk/docs]